website/blog/2022-11-31-simple-2.mdx

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
Is there a reason why date is so back in the past ?

Reply:
creation date updating it haha